cmp/resizable/Resizable.js

```diff
@@ -84,12 +79,12 @@ export class Resizable extends Component {
             items.push(this.renderCollapsedChild());
         }
 
-        if (isCollapsible) {
+        if (isCollapsible !== false) {
             const collapser = this.getCollapser();
             items = (isContentFirst ? [...items, collapser] : [collapser, ...items]);
```

Comment:
If we are going to use the getter pattern, the default needs to be in the getter!